lib/atmospheric.rb

```diff
@@ -3,7 +3,16 @@
 require_relative "atmospheric/version"
 require_relative "atmospheric/isa"
 require_relative "atmospheric/export"
+require 'lutaml/model'
+require 'lutaml/model/xml_adapter/nokogiri_adapter'
+require 'lutaml/model/json_adapter/standard_json_adapter'
+require 'lutaml/model/yaml_adapter/standard_yaml_adapter'
 
+Lutaml::Model::Config.configure do |config|
+  config.xml_adapter = Lutaml::Model::XmlAdapter::NokogiriAdapter
+  config.yaml_adapter = Lutaml::Model::YamlAdapter::StandardYamlAdapter
+  config.json_adapter = Lutaml::Model::JsonAdapter::StandardJsonAdapter
+end
```

Comment:
```suggestion
require 'lutaml/model'

Lutaml::Model::Config.configure do |config|
  config.xml_adapter_type = :nokogiri
end

```

Do we need the `:nokogiri` adapter here? We are not parsing any XML yet.

Reply:
Done